core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -703,13 +710,13 @@ private[spark] class BlockManager(
    *
    * @return true if the block was stored or false if an error occurred.
    */
-  def putBytes(
+  def putBytes[T: ClassTag]( // TODO(josh)
```

Comment:
Yes. During the original design I wasn't sure whether I wanted this to be an implicit ClassTag, but in this case it's actually desired / fine.